generator/src/test/java/com/scottlogic/deg/generator/smoke_tests/ExampleProfilesTests.java

```diff
@@ -35,7 +38,7 @@
     private static final DecisionTreeWalkerFactory walkerFactory = new SmokeTestsDecisionTreeWalkerFactory();
 
     @TestFactory
-    Collection<DynamicTest> shouldGenerateAsTestCasesWithoutErrors() throws IOException {
+    Collection<DynamicTest> shouldGenerateAsTestCasesWithoutErrors() throws IOException, InvalidProfileException {
```

Comment:
See my comments in the `DecisionTreeDataGenerator` about separation of concerns, this highlights my concern of the reverse happening.